addie/processing/idl/run_sum_scans.py

```diff
@@ -22,7 +30,14 @@ def set_sum_scans_script(self):
 
     def run(self):
         self._background = self.collect_background_runs()
-        self._runs = self.collect_runs_checked()
+        collect_return = self.collect_runs_checked()
+        self._runs = collect_return[0]
+        self._runs_name = collect_return[1]
+        self._sam_formula = collect_return[2]
+        self._mass_den = collect_return[3]
+        self._radius = collect_return[4]
+        self._pack_frac = collect_return[5]
+        self._geom = collect_return[6]
```

Comment:
See comment below about the change from a dict to a list return type. May not need all these?

Reply:
see comment above. Try to change the data structure here to dictionary.